CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+## v0.2.0 (2017-06-16)
+
+* Default listen port changed to 9216.
+* All log messages now go to stderr. Logging flags changed.
+* Fewer messages on default INFO logging level.
```

Comment:
I think it was @delgod who asked to log less. 😄 I changed logging to default levels for other Prometheus exporters. There are still should be enough information to track errors.